blocks/log.py

```diff
@@ -245,3 +254,18 @@ def __iter__(self):
 
     def get_status(self):
         return self._status
+
+    def to_dataframe(self):
+        return DataFrame.from_dict(self._storage, orient='index')
+
+
+def to_dataframe(log):
```

Comment:
No really good reason for that, but how would keep the header that checks pandas availability? Would have to implement that with a publlc-private method pair, like `to_dataframe` and `_to_dataframe`. But I guess it is not such a big deal, because if the person is brave enough to write a new log class, he should be able to work with such patterns.